README.md

```diff
@@ -1,215 +1,151 @@
-<a href="http://www.methodscount.com/?lib=com.ajts.androidmads.SQLite2Excel%3Alibrary%3A1.0.2"><img src="https://img.shields.io/badge/Methods and size-core: 130 | deps: 22949 | 16 KB-e91e63.svg"/></a>
-## Sample App
-The sample app in the repository is available on Google Play:
-
-<a href='https://play.google.com/store/apps/details?id=com.ajts.androidmads.sqlite2xlDemo&utm_source=AndroidMads&utm_campaign=AndroidMads&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' width='150px' src='https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png'/></a>
+# Source
+Inspired by [androidmads/SQLite2XL](https://github.com/androidmads/SQLite2XL) - version v1.0.2
 
 ## Features
-### 1.0.3
-Added Pull Request from https://github.com/cleathley
-1. Added the ability to provide a custom formatter  to the export.
-2. Added the ability to exclude columns from the export.
-3. Added `pretty` name mapping to export
-### 1.0.2
-1. Added support to add new column from excel while importing, if the column is not exists.
-### 1.0.1 
-1. Added Functionality to Import Excel into SQLite Database.
-2. Added Functionality to Export Blob into Image.
-3. Added Functionality to Export List of tables specified.
-## How to Download
-add the following library in your app level gradle file
+1) This library provides user the functions to export SQLite Database to Excel and import Excel data to SQLite database.
+2) This sample application has two buttons, user can choose SQLite 2 Excel and Excel 2 SQLite. 
+3) When the user clicks on the button SQLite 2 Excel, user gets to add the name and phone number of a person.
+4) User has to save the entered data by clicking the Save button, the data which is saved is displayed on the screen.
+5) User has to click on Export button to export the data to the desired xml file.
+6) Then User has to click on Excel 2 SQLite button and import the data from the xml file to database.
+7) User can now click on SQLite 2 Excel to check if the data is imported.
+8) Export with Exclude button will export the data to another xml file excluding the contact number entered by the user.
+
+## Exporting from SQLite to Excel
+<img src="screenshots/sqlitetoexcel.gif" width="500">
+
+## Importing from Excel into Database
+<img src="screenshots/exceltosqlite.gif" width="500">
+
+## Dependency
+1. For using library module in sample app, include the source code and add the below dependencies in entry/build.gradle to generate hap/support.har.
 ```groovy
-implementation 'com.ajts.androidmads.SQLite2Excel:library:1.0.4'
-```
-## How to Use
-#### The steps to use this Library
-```xml
-<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar', '*.har'])
+    implementation project(path: ':library')
```

Comment:
This library name will be problematic during maven release. Change & refactor the library name to "sqlite2xl" in all the places.

Reply:
Done